csharp/src/Ice/Communicator.cs

```diff
@@ -222,15 +242,19 @@ internal CancellationToken CancellationToken
             = new ThreadLocal<SortedDictionary<string, string>>();
         private volatile IReadOnlyDictionary<string, string> _defaultContext =
             ImmutableSortedDictionary<string, string>.Empty;
+        private volatile ImmutableList<InvocationInterceptor> _defaultInvocationInterceptors =
```

Comment:
Should this just be `IReadOnlyList<InvocationInterceptor>` instead?

Reply:
I feel it's cleaner to use the concrete type for this field. I just fixed _context to do the same for consistency.